.travis.yml

```diff
@@ -1,14 +1,18 @@
 before_install:
   - sudo apt-get install graphviz
-  
+
 language: python
 
 cache: pip
 
 python:
   - "2.7"
 
-install:
-  - pip install -r requirements-dev.txt
-
-script: "py.test -c pytest_ci.ini"
+matrix:
```

Comment:
what's matrix?

Reply:
https://docs.travis-ci.com/user/customizing-the-build/#build-matrix

It allows you to run multiple jobs in parallel.

See https://travis-ci.org/EuroPython/epcon/builds/409527705 (tests for this PR) vs https://travis-ci.org/EuroPython/epcon/builds/409527705 (tests for default branch)